docs/source/build_a_lint_rule.ipynb

```diff
@@ -130,6 +130,50 @@
     "Iteration order of attributes is the same as the order they appear in the source code.\n",
     "In this case, that means visit_If_test is called before visit_If_body and visit_If_orelse.\n",
     "\n",
+    "Use fixit's cli to generate a skeleton of adding a new rule file::\n",
+    "\n",
+    "    $ python -m fixit.cli.add_new_rule # Creates new_rule.py at fixit/rules/new_rule.py\n",
+    "    $ python -m fixit.cli.add_new_rule --path fixit/rules/my_rule.py # Creates rule file at path specified\n",
+    "\n",
+    "This will generate a model rule file used to create new rule."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import libcst as cst\n",
+    "import libcst.matchers as m\n",
+    "\n",
+    "from fixit import CstLintRule, InvalidTestCase as Invalid, ValidTestCase as Valid\n",
+    "\n",
+    "class Rule(CstLintRule):\n",
+    "    \"\"\"\n",
+    "    docstring or new_rule description\n",
+    "    \"\"\"\n",
+    "\n",
+    "    MESSAGE = \"Enter rule description message\"\n",
+    "\n",
+    "    VALID = [Valid(\"'example'\")]     # Valid examples\n",
+    "\n",
+    "    INVALID = [Invalid(\"'example'\")] # Invalid examples\n",
+    "    ..."
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "Use above structure to create a new rule. \n",
+    "\n",
```

Comment:
This looks good now.
Sorry for not thinking of it earlier. Do you mind make one more change?
The best way to demonstrate the command is to actually run it in the tutorial instead of hardcoded the result. So the output will be automatically generated and updated whenever the source code changes.

Can you update this doc to actually run the command to add a new rule?
You can run the command and just cat its content by.
```
! python -m fixit.cli.add_new_rule --path fixit/rules/my_rule.py
! cat fixit/rules/my_rule.py
```
https://github.com/Instagram/Fixit/blob/master/docs/source/getting_started.ipynb
https://fixit.readthedocs.io/en/latest/getting_started.html

Reply:
No worries, I'll add it.